### PR TITLE
Add Mac Catalyst (arm64) support

### PR DESCRIPTION
**Pull request by vitalyster**

### CODE REVIEW

**Comment by Neustradamus** [comment]
@x2on: Can you look this PR?

**Comment by x2on** [comment]
Thanks Looks good, I have to check why the CI tests are not running

**Review by x2on — APPROVED** [review]

**Comment by vgritsenko** [comment]
@x2on Hey, I see it can be approved a few months ago, can it be merged in somehow?

File: config/20-ios-tvos-cross.conf [hunk 1]
```diff
@@ -47,6 +47,18 @@ my %targets = ();
         sys_id           => "MacOSX",
     },
 
+    ## Catalyst (arm64)
+    # Based on 10-main.conf: ios-cross
+    "mac-catalyst-arm64" => {
+        inherit_from     => [ "darwin-common", "mac-catalyst-base", asm("aarch64_asm") ],
+        cflags           => add("-arch arm64"),
+        cflags           => add("-target arm64-apple-ios14.0-macabi"),
```
Comment (vgritsenko) [inline config/20-ios-tvos-cross.conf:55]: FYI: Line 54 has no effect, line 55 overrides it. You can combine both in one line (or remove `-arch` line and rely to `-target` alone).

```
       cflags           => add("-arch arm64", "-target arm64-apple-ios14.0-macabi"),
```

**Pull request merged**